docs/source/index.rst

```diff
@@ -23,14 +23,13 @@ Driver Configuration
 ====================
 
 The DNP3 driver configuration file follows `Driver Configuration File <https://volttron.readthedocs.io/en/main/agent-framework/driver-framework/platform-driver/platform-driver.html?highlight=driver_config#driver-configuration-file>`_ convention.
```

Comment:
Don't link to pages on readthedocs. Use this (note we are refering to it also as "device, not driver, configuration since driver could also refer to the the agent level config"):

`Device Configuration File <Device-Configuration-File>`_